app/models/messaging_preference.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+require 'common/models/base'
+# Secure Messaging Notification Preference Model
+class MessagingPreference < Common::Base
+  FREQUENCY_UPDATE_MAP = {
+    'none' => 0,
+    'each_message' => 1,
+    'daily' => 2
+  }.freeze
+
+  FREQUENCY_GET_MAP = {
```

Comment:
@patrickvinograd I think is probably a reasonable approach here as MHV should probably communicate to us if there are any future changes to these, but I wonder if maybe it would be best to just expose these constants via the endpoint in question? Perhaps we could cache that request in the future in a 24 hour cache or something. Thoughts?